onionshare_gui/onionshare_gui.py

```diff
@@ -317,6 +367,7 @@ def reload_settings():
                         self.timer.start(500)
                     self.share_mode.on_reload_settings()
                     self.receive_mode.on_reload_settings()
+                    self.website_mode.on_reload_settings()
                     self.status_bar.clearMessage()
 
             # If we switched off the auto-stop timer setting, ensure the widget is hidden.
```

Comment:
In this section that beings 'If we switched off the auto-stop timer setting'.... we have this block of code:

```
            # If we switched off the auto-stop timer setting, ensure the widget is hidden.
            if not self.common.settings.get('autostop_timer'):
                self.share_mode.server_status.autostop_timer_container.hide()
                self.receive_mode.server_status.autostop_timer_container.hide()
            # If we switched off the auto-start timer setting, ensure the widget is hidden.
            if not self.common.settings.get('autostart_timer'):
                self.share_mode.server_status.autostart_timer_datetime = None
                self.receive_mode.server_status.autostart_timer_datetime = None
                self.share_mode.server_status.autostart_timer_container.hide()
                self.receive_mode.server_status.autostart_timer_container.hide()
```

You will need to add an equivalent section for the `self.website_mode.server_status.autostart_timer and related attributes. Otherwise, if autostop or autostart timer is disabled in the Settings dialog whilst using Website Mode, the timers still appear in the GUI.